src/hotspot/share/oops/oop.hpp

```diff
@@ -32,6 +32,8 @@
 #include "oops/metadata.hpp"
 #include "runtime/atomic.hpp"
 #include "utilities/macros.hpp"
+#include "utilities/globalDefinitions.hpp"
```

Comment:
Nit: not included in alphabetic order (and it will include macros.hpp itself anyway).

Reply:
Fixed the order.  In the interest of "include what you use" I left macros.hpp in the list.